hw3/ThronesAPI/src/CharacterPage.jsx

```diff
@@ -0,0 +1,54 @@
+import React, { useEffect, useState } from "react";
+import Searchbar from "./Searchbar";
+import { Container, Card } from "react-bootstrap";
+import axios from "axios";
+
+const url = "https://thronesapi.com/api/v2/Characters";
+
+const CharacterPage = () => {
+  const [characters, setCharacters] = useState([]);
+  const [searchResults, setSearchResults] = useState([]);
+
+  useEffect(() => {
+    const fetchCharacters = async () => {
+      try {
+        const response = await axios.get(url);
+        const apiData = response.data;
+        setCharacters(apiData);
+      } catch (error) {
+        console.log(error);
+      }
+    };
```

Comment:
Nice way of using axios to fetch data. Error handling would feel complete if we add a message to it.

Reply:
Got it, thanks for suggesting.